bittensor/utils/version.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional
+from pathlib import Path
+import time
+
+import bittensor
+import requests
+
+VERSION_CHECK_THRESHOLD = 86400
+
+
+def _get_version_file_path() -> Path:
+    return Path.home() / ".bittensor" / ".version"
```

Comment:
This name is so confusing! I think the file should be called smth like `.last_known_version` and variable using it should be renamed to `last_known_version`.

Reply:
Also, why is it a function and not a constant?

Reply:
The idea was to avoid a filesystem call to boost cold start. I was also not sure about how things will work on Windows, so having a dedicated function for that, can make it easier to update. But probably a bit premature

Reply:
This makes sense now, thanks for the explanation!